hack/update/update_all/update_all.go

```diff
@@ -65,8 +65,7 @@ func main() {
 		updateCmd.Stderr = os.Stderr
 
 		if err := updateCmd.Run(); err != nil {
-			log.Printf("Failed to update %s: %v", component, err)
-			continue
+			log.Fatalf("Failed to update %s: %v", component, err)
```

Comment:
This stop on the first error, which is less useful for reporting multiple updates.